src/plugins/data_views/docs/openapi/components/examples/update_field_metadata_request.yaml

```
@@ -1,9 +1,14 @@
-summary: Set popularity count for field foo.
+summary: Update multiple metadata fields.
```

```suggestion
summary: Update metadata for multiple fields.
```
I interpret the current summary as updating "metadata fields" instead of updating metadata for the specified fields. I wonder if something like this would be clearer?